File: src/modules/common/infrastructure/consts/goTEnums.ts
export enum HouseType {
  Lannister = "Lannister",
  Baratheon = "Baratheon",
  Stark = "Stark",
  Greyjoy = "Greyjoy",
  Martell = "Martell",
  Tyrell = "Tyrell"
}

export enum PlayerStatus {
  Joining = "0",
  Joined = "1",
  Ready = "2"
}

export enum GameStatus {
  WaitingForPlayers,
  Started,
  Finished
}

export enum WinCondition {
  Castles = "Castles",
  Points = "Points"
}

export enum CastleType {
  None = "None",
  Large = "Large",
  Small = "Small"
}

export enum FieldType {
  Land = "Land",
  Sea = "Sea",
  Port = "Port",
  River = "River"
}

File: server/GotGame.RestServer/Infrastructure/Consts/GotEnums.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GotGame.RestServer.Infrastructure.Consts
{
    public enum House
    {
        Lannister,
        Baratheon,
        Stark,
        Greyjoy,
        Martell,
        Tyrell
    }

  public enum PlayerStatus
  {
    Joining,
    Joined,
    Ready
  }

  public enum WinCondition
  {
    Castles,
    Points
  }
}
